frontend/scripts/init-dynamodb.sh

```diff
@@ -101,4 +107,20 @@ aws dynamodb put-item --table-name HadithCorrectionsArchive --item '{
    "moderatedBy":{
       "S":"rootuser"
    }
-}' --endpoint-url http://dynamodb-local:8000 --region us-west-2
\ No newline at end of file
+}' --endpoint-url http://dynamodb-local:8000 --region us-west-2
+
+aws dynamodb put-item --table-name Users --item '{
+    "username":{
+        "S": "ahsan"
+    },
+    "permissions":{
+        "M":{
+              "manage users": {
```

Comment:
can we replace the space with an underscore?

Reply:
done